index.html

```diff
@@ -107,6 +109,21 @@ <h2>Status</h2>
 				</div>
 			</article>
 
+			<article>
+				<h2>Form</h2>
+				<ts-label>Text</ts-label>
+				<ts-input id="test_input">
+					<input />
+				</ts-input>
+				<!-- <script>
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this commented out code. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.ts.sv/coding_rules#rule_key=Web%3AAvoidCommentedOutCodeCheck)

Reply:
Can you please remove this?